package.json

```diff
@@ -77,7 +81,10 @@
       "raf/polyfill",
       "./webpack/test_setup.js"
     ],
-    "setupTestFrameworkScriptFile": "./webpack/global_test_setup.js",
+    "setupFilesAfterEnv": [
```

Comment:
Deprecation change and also this now takes multiple arguments which is needed for react-testing-library to load jest-dom here as opposed to each file